script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -47,13 +47,13 @@ Anyways, that was something like a fluffy whip that stimulated the yearnings of
 Immediately, a storm of questions erupts from the boys.
 These topics are usually discussed in an anonymous forum, but the guys who were hit by her wink couldn't even wait for that.
 <Page14>
-The girls look at them coldly as if saying, 'Boys are pathetic,' but it seems that the scorn of classmates who they are tired of looking at are not important to them as they are in the heat of the moment.
+The girls look at them coldly as if saying, 'Boys are pathetic,' but it seems that the scorn of classmates who they are tired of looking at are not important to them as this is in the heat of the moment.
```

Comment:
Revert - the boys are lost in the heat of the moment, and that is why they don't care about the opinion of the girls